src/plugins/persistence/couch/package-lock.json

```diff
@@ -0,0 +1,1421 @@
+{
+    "name": "openmct-couch-plugin",
+    "version": "1.0.0",
```

Comment:
@ozyx @akhenry broke this out into a separate discussion https://github.com/nasa/openmct/issues/7847